packages/lib-classifier/src/plugins/tasks/models/Task.js

```diff
@@ -1,32 +1,33 @@
+import cuid from 'cuid'
 import { getRoot, types } from 'mobx-state-tree'
 import Annotation from './Annotation'
 
+
 const Task = types.model('Task', {
+  // override annotation in individual task models with specific annotation types
+  annotation: types.maybe(types.safeReference(Annotation)),
   taskKey: types.identifier,
   required: types.maybe(types.boolean),
   type: types.literal('default')
 })
   .views(self => ({
-    get annotation () {
-      const { classifications } = getRoot(self)
-      const { annotation } = classifications || {}
-      const currentAnnotation = annotation ? annotation(self) : self.defaultAnnotation
-      return currentAnnotation
-    },
```

Comment:
Note to self: since this PR removes the `.annotations` view from the general Task model, it would mean that the _Task model_ is no longer responsible for _initialising its own default annotation._

(Unless I'm missing some hidden initialisation with `annotation: types.maybe(types.safeReference(Annotation))`)

Which might mean that it's _theoretically possible_ for `myTask.annotation` to return null or undefined (or something equally wonky) if the _new_ "initialising a default annotation" step goes awry.

So: the responsibility for "initialising" a default annotation value has been moved to `src/components/Classifier/components/TaskArea/components/Tasks/Tasks.js`, so check this bit to see if it's possible to screw this up.

- Can the annotation initialiser (`const annotation = addAnnotation(task) ; task.setAnnotation(annotation)`) be called _0 times?_ (Thoughts: mmm... probably not. Would there ever be a scenario where you want to have an instance of a Task _model..._ but not want to render the Task _component?_ )

- Can the annotation initialiser be called _multiple times,_ causing multiple default annotations? (Thoughts: uh... what can cause the `Tasks` component to re-render? Multi-Task, multi-Step workflows tested fine in early tests, but what if you go back/forth between Tasks?)

Reply:
`addAnnotation(task, value)` is really badly named. It doesn't add a new annotation for `task` if one already exists: it replaces the existing one. In fact, the classification can only have one annotation per task. There's an issue tracking this: #1226.

Reply:
If you don't specifically pass an annotation to `myTask`, with `myTask.setAnnotation`, then `myTask.annotation` will be undefined and the task won't be able to do any work.

Reply:
Thanks for the clarification 👌 The fact that `addAnnotation()` _overwrites_ existing annotations makes me feel better about the "multiple added annotations" (now a non-)issue, but having only one annotation per task is its own thing.

Anyway, I tried tracing when `ClassificationStore.addAnnotation` gets triggered when navigating on different kind of Workflows and everything _looks_ legit, though I'm still scratching my head as to the _'how'_ things work in some cases. (e.g. _"Wait, how is ClassificationStore.addAnnotation() called with a non-undefined 2nd argument/annotationValue? I definitely see it happening when I complete the current active Task and submit/move to the next task, but I don't see where in the code has ClassificationStore.addAnnotation(task, someNonEmptyValue)")_

Not worth discussing my thoughts at the moment though, as I still need to digest, and this PR still needs to get merged soon.